### PR TITLE
added in new torchvision transformer models

### PR DESCRIPTION
**Pull request by ethanwee1**
Adding in new transformer models for micro benchmarking.

### CODE REVIEW

**Review by ramcherukuri — COMMENTED** [review]
@ethanwee1, can you add an output summary of the test run?

**Comment by ethanwee1** [comment]
Script used: 
```
python3 micro_benchmarking_pytorch.py --device_ids=0 --network efficientnet_b0 --batch-size 256 --iterations 25 --fp16 1 --distributed_dataparallel --rank 0 --world-size 2 --dist-backend nccl --dist-url tcp://127.0.0.1:4332 &
python3 micro_benchmarking_pytorch.py --device_ids=1 --network efficientnet_b0 --batch-size 256 --iterations 25 --fp16 1 --distributed_dataparallel --rank 1 --world-size 2 --dist-backend nccl --dist-url tcp://127.0.0.1:4332 &
```
Output: 

```

Rendezvous complete. Created process group...
INFO: Rank 1 running distributed_dataparallel on devices: [1]
Rendezvous complete. Created process group...
INFO: Rank 0 running distributed_dataparallel on devices: [0]
INFO: running forward and backward for warmup.
INFO: running forward and backward for warmup.
INFO: running the benchmark..
INFO: running the benchmark..
OK: finished running benchmark..
--------------------SUMMARY--------------------------
Microbenchmark for network : efficientnet_b0
--------This process: rank 0--------
Num devices: 1
Dtype: FP16
Mini batch size [img] : 256
Time per mini-batch : 0.9076788997650147
Throughput [img/sec] : 282.0380644149322

OK: finished running benchmark..
--------Overall (all ranks) (assuming same num/type devices for each rank)--------
Num devices: 2
--------------------SUMMARY--------------------------
Dtype: FP16
Mini batch size [img] : 512
Microbenchmark for network : efficientnet_b0
Time per mini-batch : 0.9076788997650147
Throughput [img/sec] : 564.0761288298644
--------This process: rank 1--------
Num devices: 1
Dtype: FP16
Mini batch size [img] : 256
Time per mini-batch : 0.9076578712463379
Throughput [img/sec] : 282.0445986420821

--------Overall (all ranks) (assuming same num/type devices for each rank)--------
Num devices: 2
Dtype: FP16
Mini batch size [img] : 512
Time per mini-batch : 0.9076578712463379
Throughput [img/sec] : 564.0891972841642

```

File: micro_benchmarking_pytorch.py [hunk 1]
```diff
@@ -87,6 +87,26 @@ def weight_init(m):
...
+        "vit_h_14":            torchvision.models.vit_h_14,
+        "efficientnet_b0" :    torchvision.models.efficientnet_b0,
+        "efficientnet_b1" :    torchvision.models.efficientnet_b1,
+        "efficientnet_b2" :    torchvision.models.efficientnet_b2,
+        "efficientnet_b3" :    torchvision.models.efficientnet_b3,
+        "efficientnet_b4" :    torchvision.models.efficientnet_b4,
+        "efficientnet_b5" :    torchvision.models.efficientnet_b5,
+        "efficientnet_b6" :    torchvision.models.efficientnet_b6,
+        "efficientnet_b7" :    torchvision.models.efficientnet_b7,
+        "maxvit_t" :           torchvision.models.maxvit_t,
```
Comment (pruthvistony) [inline micro_benchmarking_pytorch.py:109]: Can you update info about this model in the confluence page - https://confluence.amd.com/display/MLSE/Transformers+based+models

**Review by pruthvistony — APPROVED** [review]

File: micro_benchmarking_pytorch.py (hunk 1 above)
Comment (ethanwee1) [reply micro_benchmarking_pytorch.py:109]: Updated @pruthvistony

**Pull request merged**